<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>599</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/img/7419b80742274fc72231166504b06394.jpeg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="login">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>300</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Marker Felt</family>
      <pointsize>24</pointsize>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(164, 164, 164);</string>
    </property>
    <property name="text">
     <string>Log in</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Subject">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>341</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Marker Felt</family>
      <pointsize>31</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(164, 164, 164);</string>
    </property>
    <property name="text">
     <string>The Covid Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AboutGame">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Marker Felt</family>
      <pointsize>24</pointsize>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(164, 164, 164);</string>
    </property>
    <property name="text">
     <string>About the game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartGame">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>460</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Marker Felt</family>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Start </string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitGame">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>460</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Marker Felt</family>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="permission_Label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>350</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 38, 0);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="backround_image">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>951</width>
      <height>601</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>backround_image</zorder>
   <zorder>login</zorder>
   <zorder>Subject</zorder>
   <zorder>AboutGame</zorder>
   <zorder>StartGame</zorder>
   <zorder>ExitGame</zorder>
   <zorder>permission_Label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
